code/modules/food_and_drinks/food/snacks_meat.dm

```diff
@@ -131,6 +131,16 @@
 	. = ..()
 	eatverb = pick("bite","chew","nibble","gobble","chomp")
 
+/obj/item/reagent_containers/food/snacks/blackpudding
+	name = "black pudding"
+	desc = "Bloody delicious."
+	icon_state = "blackpudding"
+	filling_color = "#5B3330"
+	bonus_reagents = list(/datum/reagent/consumable/nutriment = 1, /datum/reagent/consumable/nutriment/vitamin = 2)
+	list_reagents = list(/datum/reagent/consumable/nutriment = 6, /datum/reagent/consumable/nutriment/vitamin = 2)
+	tastes = list("savoryness" = 1)
+	foodtype = MEAT | BREAKFAST
```

Comment:
why does this count as meat? if it has meat in it, add the meat taste

Reply:
should also have onion taste

Reply:
this is already included when you bite a black pudding. savoryness is just an extra taste.

Reply:
gotcha